src/applications/my-education-benefits/config/form.js

```diff
@@ -36,9 +36,11 @@ import PhoneReviewField from '../components/PhoneReviewField';
 import DateReviewField from '../components/DateReviewField';
 import EmailReviewField from '../components/EmailReviewField';
 
+import environment from 'platform/utilities/environment';
+
 import {
-  activeDutyLabel,
-  selectedReserveLabel,
+  chapter30Label,
```

Comment:
I changed this labels to have meaningful names. It was hard to track that they were related to chapter30 and chapter 1606 relinquisments.